docs/specs/oidc.md

```diff
@@ -460,29 +458,22 @@ Content-Type: application/json
 {"result":{"session_token":"<session token>"}}
 ```
 
-A one-time-use session token would be returned, and the native app may
-then use it in OAuth authorization flow to copy user session to web user
-agent:
+A session token would be returned, and the native app may inject it into the
+session token cookie in web view/header, and open the desired page:
 ```
-GET /oauth2/authorize?client_id=client_id&prompt=none&response_type=none
-    &login_hint=https%3A%2F%2Fauthgear.com%2Flogin_hint%3Ftype%3Dsession_token%26session_token%3D<session token>
-    &redirect_uri=<redirect URI> HTTP/1.1
+GET /settings HTTP/1.1
```

Comment:
Let's keep the original approach by using `/oauth2/authorize` to cause the server to write a valid cookie. The developer can also use `redirect_uri` to open a specific settings page.